src/htmxl/compose/write/elements.py

```diff
@@ -225,3 +229,21 @@ def create_datavalidation(element, writer, styler, style):
     validation = DataValidation(type="list", formula1=f'"{validation_formula}"', allow_blank=True)
     validation.hide_drop_down = False
     writer.add_validation(element_id, validation)
+
+
+def _write_default_value(element, writer, styler, style, list_validation):
+    with writer.record() as recording:
+        data_type = _type_map[element.attrs.get("data-type")]
+        list_default_value = element.attrs.get("value", "")
+        data_validations = writer.data_validations
+        allowed_values = data_validations[list_validation].formula1.replace('"', "").split(",")
+
+        if writer.data_validations[list_validation].allow_blank:
+            allowed_values.append("")
+
+        if list_default_value not in allowed_values:
+            raise DataValidationError(element, list_default_value, allowed_values)
```

Comment:
`openpyxl` doesn't validate data itself. We need to this^ manually.
```python
>>> wb = Workbook()
>>> ws = wb.active
>>> dv = DataValidation(type="list", formula1='"Dog,Cat,Bat"', allow_blank=True)
>>> ws.add_data_validation(dv)
>>> dv.add(ws["A1"])
>>> ws["A1"].value = "InvalidValue"
>>>
>>> # No Error^^
```

Reply:
i'm not sure that i'm so concerned about the validation actually occuring during template writing. At least personally.

My take is that this library is much more directly oriented towards producing an excel file of specific shape/behavior.

Reply:
With that being said, i'm not exactly opposed to this specific check, because it's something that we **can** realistically check for. Just saying so, because plenty of formulas or examples of features like this **wont** be easily implementable without trying to emulate excel itself.

Reply:
...actually i wrote all that on the main PR page before i actually read the impl above.

I think maybe i'd prefer to avoid doing this check actually? It's pretty unfortunate that we have to jump around after the fact, and re-parse the formula back out. And it's in a location that we cannot generally assume this to be true. This will only work for a specific kind of formula.